src/System-Localization/String.extension.st

```diff
@@ -1,13 +0,0 @@
-Extension { #name : 'String' }
-
-{ #category : '*System-Localization' }
-String >> translated [
-"answer the receiver translated to the default language"
-	^ NaturalLanguageTranslator translate: self
-]
-
-{ #category : '*System-Localization' }
-String >> translatedTo: localeID [
-	"answer the receiver translated to the given locale id"
-	^ NaturalLanguageTranslator translate: self toLocale: localeID
-]
```

Comment:
Maybe these extensions should be kept as deprecated just returning self for people using them